### PR TITLE
completed 16 of 16 files

### PR DESCRIPTION
**Pull request by clintKunz**

### CODE REVIEW

File: src/printing.py [hunk 1]
```diff
@@ -11,7 +11,11 @@
 # Using the printf operator (%), print the following feeding in the values of x,
 # y, and z:
 # x is 10, y is 2.25, z is "I like turtles!"
```
Comment (koennecker) [inline src/printing.py:13]: The problem is asking you to round y to the second decimal. Check out this link for the first two https://mkaz.blog/code/python-string-format-cookbook/ and this one for the third https://stackoverflow.com/questions/45310254/fixed-digits-after-decimal-with-f-strings

File: src/lists.py [hunk 2]
```diff
@@ -8,22 +8,29 @@
...
 print(x)
 
 # Using y, change x so that it is [1, 2, 3, 4, 8, 9, 10]
 # YOUR CODE HERE 
+x.extend(y)
 print(x)
 
 # Change x so that it is [1, 2, 3, 4, 9, 10]
 # YOUR CODE HERE 
+del x[4]
```
Comment (koennecker) [inline src/lists.py:21]: Interesting choice. You could also do `x.remove(8)` to remove the first occurrence of 8, or `x.pop(4)` to remove the fourth index.

File: src/slices.py [hunk 3]
```diff
@@ -12,26 +12,26 @@
 a = [2, 4, 1, 7, 9, 6]
 
 # Output the second element: 4:
-print()
+print(a[1:2])
 
 # Output the second-to-last element: 9
-print()
+print(a[-2:-1])
```
Comment (koennecker) [inline src/slices.py:18]: `a[-2]` would also work.

**Comment by koennecker** [comment]
Great start on the first day of the project, Clint!

**Comment by clintKunz** [comment]
There is a 'None' being printed in the cal file, and I'm not sure why

**Comment by koennecker** [comment]
@clintKunz the `prmonth` method prints to the terminal itself and returns `None` if you remove the `print` statement from your function call at the bottom, that will remove the `None`. Though, instead of returning "invalid input, should be [month] [year]" in your get_date function you would need to just print directly from the else statement for the error message.

**Comment by clintKunz** [comment]
Oh okay, thanks for the feedback!

File: src/scopes.py [hunk 4]
```diff
@@ -19,12 +20,13 @@ def outer():
     y = 120
 
     def inner():
+        global y
```
Comment (koennecker) [inline src/scopes.py:23]: This works, but there's a way to do it with the `nonlocal` keyword while keeping the print statement inside the function https://www.w3schools.com/python/ref_keyword_nonlocal.asp

**Comment by koennecker** [comment]
Everything looks good. Great job with the frequent and descriptive commits.

**Pull request merged**